projects/dashboards/gfbio/src/app/help/help.component.html

```diff
@@ -1,11 +1,18 @@
 <geoengine-sidenav-header>Help</geoengine-sidenav-header>
 
-<p>
-    <span class="vat">Vat</span> is developed by the Database Research Group of the University of Marburg (head: Prof. Bernhard Seeger) in
-    cooperation with the Senckenberg Biodiversity and Climate Research Centre (BiK-F) (head: Prof. Thomas Hickler). You can contact us at
-    <a href="mailto:vat@mathematik.uni-marburg.de" target="_blank">vat@mathematik.uni-marburg.de</a> or visit our
-    <a href="https://github.com/umr-dbs/VAT " target="_blank">GitHub</a> for further information.
-</p>
+<div class="attribution">
+    <p>
+        <span class="vat">Vat</span> is developed by the Database Research Group of the University of Marburg (head: Prof. Bernhard Seeger)
+        in cooperation with the Senckenberg Biodiversity and Climate Research Centre (BiK-F) (head: Prof. Thomas Hickler). You can contact
+        us at <a href="mailto:vat@mathematik.uni-marburg.de" target="_blank">vat@mathematik.uni-marburg.de</a> or visit our
+        <a href="https://github.com/umr-dbs/VAT " target="_blank">GitHub</a> for further information.
```

Comment:
This info and repo are outdated.

Reply:
the repo is outdated, true. But what about the rest? It's still devleoped/maintained by agdbs right? Do you want me to remove Senckenberg as well?

Reply:
The instance is maintained by AGDBS, right.

But I would link to the new repo and I'm not sure that this e-mail works.

I'm also not sure if BiK-F needs still to be there, maybe there could be some kudos that they helped in the design.

Reply:
The E-Mail works. It somehow lands in my Uni mailbox. But maybe we should link to the gfbio.org impressum instead?

Reply:
Ok

Reply:
4e57f06720143f30f3f3caaeaddc3c630d8f8d35